providers/curation/github.js

```diff
@@ -93,14 +93,23 @@ class GitHubCurationService {
         )
       )
     )
-    return (userGithub || serviceGithub).pullRequests.create({
+    const result = await (userGithub || serviceGithub).pullRequests.create({
       owner,
       repo,
       title: prBranch,
       body: patch.description,
       head: `refs/heads/${prBranch}`,
       base: branch
     })
+    const number = result.data.number
+    const comment = {
+      owner,
+      repo,
+      number,
+      body: `You can review the change introduced to the full definiton at [ClearlyDefined](https://clearlydefined.io/view_pr/${number}).`
```

Comment:
this url changed. It is now 'contributions'

also, type in 'definition' (missing the last i)

Reply:
Note that I merged this PR and fixed the above issue